index.js

```diff
@@ -79,6 +80,12 @@ const register = (server, options) => {
       if (statusCode === 404) {
         return;
       }
+      // if ignoreUnauthorizedTry is true then don't report 401's that have 'try' mode
+      if (options.ignoreUnauthorizedTry && statusCode === 401) {
+        if (request.route.settings.auth && request.route.settings.auth.mode === 'try') {
```

Comment:
I think you can just check the tags that are sent back from hapi

Reply:
I don't see a difference between the tags emitted for 'try' and the other tags.